etl_server/pipelines/controllers.py

```diff
@@ -139,8 +140,8 @@ def query_pipelines(self, user=None, public=None):
             status = dict(status='didnt-run')
             status = statuses.get(res['key'], status)
             res['value']['status'] = status
-        query_results['result'] = list(map(lambda x: dict(x.get('value'), owner=x.get('owner'), private=x.get('private')), results))
-
+        query_results['result'] = list(map(lambda x: dict(x.get('value'), owner=x.get('owner'), private=x.get('private'), 
+                                    created_at=(x['created_at']).isoformat(), updated_at=(x.get('updated_at')).isoformat()), results))
```

Comment:
Why the difference between the two properties? For one you're using square brackets [] and for the other you're using get().
Also it would probably be better to check that the value is not None before calling isoformat() on it.